_data/changelogs/component-card.yml

```diff
@@ -2,6 +2,13 @@ title: Card
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that caused the component to ignore the `$theme-card-font-family` setting.
+    affectsStyles: true
```

Comment:
**question:** Do we want to mark this as a breaking change since this has the risk of changing the card font family? 
```suggestion
    isBreaking: true
    affectsStyles: true  
```

Reply:
I will say, it feels like the risk of change is rather low. Users would have to have previously set `$theme-card-font-family`, which would have previously not changed anything, and then keep the setting for it to have any "unexpected" settings. 

That said, happy to call this out if we're worried about it!